src/OrchardCore/OrchardCore.Users.Abstractions/Services/IUserService.cs

```diff
@@ -60,6 +61,8 @@ public interface IUserService
         /// <returns>The <see cref="IUser"/> represents the retrieved user.</returns>
         Task<IUser> GetUserAsync(string userName);
 
+        Task<IEnumerable<IUser>> GetUsersAsync();
```

Comment:
I would not expose it in the service. It's a bad method (unbounded).

Reply:
I saw similar thing in `Roles`, I will try to remove this for now

Reply:
I thought about it, but the probability of having that many roles is smaller